python_modules/dagster/dagster/_grpc/protos/api.proto

```diff
@@ -168,3 +170,12 @@ message StartRunReply {
 message GetCurrentImageReply {
   string serialized_current_image = 1;
 }
+
+message ExternalJobRequest {
+  string serialized_repository_origin = 1;
+  string job_name = 2;
+}
+
+message ExternalJobReply {
+  string serialized_response = 1;
+}
```

Comment:
per feedback on other diff - could imagine having two fields, one for good, one for error, vs. parsing the string looking for clues that it's a serialized error - not the end of the world